core/app/components/FormBuilder/FormBuilder.tsx

```diff
@@ -95,49 +96,12 @@ const elementPanelTitles: Record<PanelState["state"], string> = {
 	editingButton: "Edit Submission Button",
 };
 
-const PanelHeader = ({ state }: { state: PanelState["state"] }) => {
```

Comment:
I moved all the side panel stuff to another component so I could reuse it in the form fill page. I think this would be better off using a drawer or maybe even the new side bar component from shadcn, but for now I reused what we had to keep things consistent